### PR TITLE
ASM-6618 enable setting multiple NTP servers/peers

### PR DESCRIPTION
**Pull request by ksw2599**

### CODE REVIEW

**Comment by ksw2599** [comment]
DON'T MERGE YET - Test in progress for Windows 2012

**Comment by ksw2599** [comment]
The test for Win2012 seems successful.

File: tasks/redhat.task/post_install.erb [hunk 1]
```diff
@@ -51,11 +51,15 @@ if [ "$ntp_server" != "" ]; then
   sed -e 's/^\(\s*server.*\)/#\1/' /etc/ntp.conf.orig > /etc/ntp.conf
 
   # Append desired server entry
-  cat >> /etc/ntp.conf << EOL
+  echo "# SERVER(S) ADDED BY ASM KICKSTART" >> /etc/ntp.conf
 
-# SERVER(S) ADDED BY ASM KICKSTART
-server $ntp_server iburst
-EOL
+<% if (options['ntp_server'] || '').include? ',' %>
```
Comment (gavin-scott) [inline tasks/redhat.task/post_install.erb]: You can remove this whole `if/else` block and just have lines 57-59. If there is no comma `split(',')` will return an array of size one with the single ntp server.

File: tasks/redhat.task/post_install.erb [hunk 2]
```diff
@@ -39,10 +39,10 @@ umount /tmp/mnt
...
 # NTP Setting
-ntp_server=<%=
+ntp_server="<%=
     options = node.policy.node_metadata['installer_options'] || {}
     (options['ntp_server'] || '').strip.size > 0 ? options['ntp_server'] : ""
-%>
+%>"
 
 ### NTP CONFIGURATIONS ###
 if [ "$ntp_server" != "" ]; then
```
Comment (gavin-scott) [inline tasks/redhat.task/post_install.erb]: This (the bash check for an empty `$ntp_server` variable that was set a couple lines above doesn't really make sense any more (if it ever did). If would remove it and do the `if` at the ERB level.

**Comment by gavin-scott** [comment]
It looks like you need to make these changes for the ESXi installer as well.

**Comment by ksw2599** [comment]
@gavin-scott can you see if it looks good now?

**Pull request merged**